packages/compartment-mapper/src/node-powers.js

```diff
@@ -146,8 +221,8 @@ const makeReadPowersSloppy = ({ fs, url = undefined, crypto = undefined }) => {
  * but `makeWritePowers` presents a type that requires `url`.
  *
  * @param {object} args
- * @param {typeof import('fs')} args.fs
- * @param {typeof import('url')} [args.url]
+ * @param {FsAPI} args.fs
```

Comment:
I have a grudge against API where Interface is sufficient. At least consider `Api` if `Interface` is too bulky. That’s my other grudge: delimited initialisms like XMLHttpRequest should either be XMLHTTPRequest or XmlHttpRequest, and one of those loses the delimiter.